### PR TITLE
Add Reader screen for unified prologue carousel

### PR DESCRIPTION
**Pull request by frosty**
Ref #15056. This PR adds the Reader screen to the prologue carousel.

| iPhone | iPad |
|---|---|
| Simulator Screen Shot - iPhone 12 Pro - 2021-03-29 at 21 53 38 | Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-03-29 at 21 55 08 |

*To test*

- Build, run and log out if necessary
- Verify that the Reader (last) page of the login prologue is consistent with the designs (see issue #15056; see also example screenshot above)

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueReaderContentView.swift [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                .font(font)
+                .bold()
+            Spacer()
+        }
+    }
+}
+
+/// A view modifier that applies style to a text to make it look like a tag token.
+///
+private struct TagItem: ViewModifier {
```
Comment (Gio2018) [inline WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueReaderContentView.swift:184]: Nice!

**Review by Gio2018 — COMMENTED** [review]
Hey @frosty , I left one code comment and found a few layout issues:

tags are not very visible in dark mode, looks like the background color did not switch:

<p align=center>

</p>

on some devices, the vertical alignment is too high or too low (see examples). We might consider positioning the container view in `UnifiedProloguePages` relative to the screen size. I tried something like this in #16187

iPad Pro 9.7 iOS 13.5 | iPhone SE 2nd gen iOS 14.4
-- | --
 |

**Comment by frosty** [comment]
@Gio2018 I've fixed the dark mode issue, but let's tackle the spacing of all the views altogether in a future PR.

**Review by Gio2018 — APPROVED** [review]
Works great in dark mode.

A few optional minor adjustments to the icons might be made based on the designs, and yes let's address the positioning separately.

**Pull request merged**